incubator/hnc/docs/releasing.md

```diff
@@ -114,16 +120,27 @@ this](https://cloud.google.com/cloud-build/docs/securing-builds/use-encrypted-se
 
 ## Test
 
-Test! At a minimum, install it onto a cluster, download the kubectl plugin, and
-run:
+Fill in the test signoff grid in the release notes. We generally test on the
+three GKE channels (rapid, regular and stable) as well as the latest KIND
+version.
+
+To test GKE, ensure you've (manually) downloaded the kubectl plugin according to
+your release instructions, set your kubecontext to point to the cluster you're
```

Comment:
Is kubecontext an official name for kubectl context?

Reply:
Dunno :D Fixed.